src/components/CourseChecklist/index.jsx

```diff
@@ -65,16 +66,17 @@ class CourseChecklist extends React.Component {
     return (
       <WrappedMessage message={message}>
         {displayText => (
-          <Icon
-            className={[classNames(
-              'm-4 col-1',
-              FontAwesomeStyles.fa,
-              FontAwesomeStyles['fa-2x'],
-              this.getCompletionIconClassNames(isCompleted)),
-            ]}
-            id={`icon-${checkID}`}
-            screenReaderText={displayText}
-          />
+          <div className="text-center">
```

Comment:
I don't remember - does this require the Paragon `React.Fragment` update to center as expected?